ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2983,7 +2983,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     TradeWorkFlowOnMarketDataDisable();
                     return ProcessWorkFlow(StrategyTradeWorkFlowState.GoOCOLongShortSubmitOrder);
                 case StrategyTradeWorkFlowState.GoOCOLongShortSubmitOrder:
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)

```

Comment:
This is required - IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode use the same paths a lot of the time-